src/scss/elements/_card.scss

```diff
@@ -0,0 +1,65 @@
+.card{
```

Comment:
What you call a `.card__item` is actually the card.

These styles for the whole page layout are not related with the Card component.